tests/SpecTests/Operation.php

```diff
@@ -101,6 +101,23 @@ public static function fromCommandMonitoring(stdClass $operation)
         return $o;
     }
 
+    public static function fromCrud(stdClass $operation)
+    {
+        $o = new self($operation);
+
+        $o->resultExpectation = ResultExpectation::fromCrud($operation, $o->getResultAssertionType());
+
+        if (isset($operation->collectionOptions)) {
+            $o->collectionOptions = (array) $operation->collectionOptions;
+        }
+
+        if (isset($operation->databaseOptions)) {
+            $o->databaseOptions = (array) $operation->databaseOptions;
+        }
```

Comment:
`databaseOptions` technically isn't defined within the CRUD spec. It's possible this could show up in the future, so I'll defer to you if you'd like to keep it. Elsewhere, I was being very pedantic and only parsing fields in the documented test format (hoping to invite a failure if something new was introduced unknowingly).

Reply:
This was shamelessly copy-pasted from `fromTransactions`. Since it's not an option of the spec as you mention, I'd suggest removing it from both methods until the time comes to add it.